src/test/resources/yaml/testTemplate.yaml

```diff
@@ -11,3 +11,6 @@ format:
   additional:
     - TitleIssueLinkIncludedCheck
     - OneCommitOnlyCheck
+
+  CI:
+    - TestCI
```

Comment:
is this necessary when you stub `getCI` method?

Reply:
CI element must be specified.

Reply:
the question that arise whether it would be possible to substitu which CIs are available for configuration in the first place. That is to change from which options user can chose CIs from in the test